tests/e2e/logging/test_openshift-logging.py

```diff
@@ -140,3 +142,51 @@ def test_create_new_project_to_verify_logging(self, create_pvc_and_deploymentcon
             logger.info("The new project exists in the EFK stack")
         else:
             raise ModuleNotFoundError
+
+    @pytest.mark.polarion_id("OCS-665")
+    @tier4
+    def test_create_and_delete_multiple_app_pods_and_verify_logs(self, multi_pvc_factory, dc_pod_factory):
+        """
+        The test is to create and delete multiple app pods in multiple project
+        to verify in EFK stack
+        """
+
+        num_of_pvcs = 2
+        pvc_size = 2
+
+        pvc_objs = multi_pvc_factory(
+            size=pvc_size, num_of_pvc=num_of_pvcs
+        )
+
+        dc_pod_objs = list()
+        for pvc_obj in pvc_objs:
+            dc_pod_objs.append(dc_pod_factory(pvc=pvc_obj))
+
+        dc_pod_list = []
+        for pod in dc_pod_objs:
+            dc_pod_list.append(pod.name)
+
+        pod_list = get_all_pods(namespace='openshift-logging')
+        elasticsearch_pod = [
+            pod.name for pod in pod_list if pod.name.startswith('elasticsearch')
+        ]
+        elasticsearch_pod_obj = get_pod_obj(
+            name=elasticsearch_pod[1], namespace='openshift-logging'
+        )
+
+        iterations = 1
+        retry_time = 300
+        while iterations < 5:
+            project_list = elasticsearch_pod_obj.exec_cmd_on_pod(
+                command='indices | grep project', out_yaml_format=True
+            )
+            logger.info(project_list)
+            if(set(dc_pod_list).issubset(set(project_list))):
+                logger.info("All the projects created exists in the EFK stack")
+                break
+            else:
+                iterations += 1
+                time.sleep(retry_time)
+
+        assert iterations < 5, "Failed"
```

Comment:
Please add some details to the assertion message

Reply:
Removed this part